README.md

```diff
@@ -105,6 +105,17 @@ To run the project, execute the following steps:
 - Twitter: [@twitterhandle](https://twitter.com/share_insider)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohammad-ali-reza-25686428b/)
 
+👤 **Collins Akorda**
+
+- GitHub: [@githubhandle](https://github.com/firstpj)
+- Twitter: [@twitterhandle](https://twitter.com/AkordaC)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/collins-akorda-bb46b2232/)
+
+👤 **Fatuma Hussein**
+
+- GitHub: [@githubhandle](https://github.com/fatmahussein)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/fatuma-hussein-48149917b)
+
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍